src/dialogs/examples/teams/UpdateCardMsgSetupDialog.ts

```diff
@@ -3,23 +3,38 @@ import { TriggerActionDialog } from "../../../utils/TriggerActionDialog";
 import { DialogIds } from "../../../utils/DialogIds";
 import { DialogMatches } from "../../../utils/DialogMatches";
 import { Strings } from "../../../locale/locale";
+let config = require("config");
 
 export class UpdateCardMsgSetupDialog extends TriggerActionDialog {
-
+    // setup the card message and then user can update the card using below update dialog file
+    // microsoft-teams-sample-complete-node\src\dialogs\examples\teams\UpdateCardMsgDialog.ts
     private static async setupCardMessage(session: builder.Session, args?: any | builder.IDialogResult<any>, next?: (args?: builder.IDialogResult<any>) => void): Promise<void> {
-        let tempCard = new builder.HeroCard(session).title(Strings.loading);
-        let msg = new builder.Message(session).addAttachment(tempCard);
+        let buttons = new Array<builder.CardAction>();
```

Comment:
There's only 1 button, no need for this array, just do .buttons([messageBackButton])

Reply:
Same above in UpdateCardMsgDialog

Reply:
Done.